app/authentication/auth.py

```diff
@@ -136,11 +136,19 @@ def requires_auth():
         raise AuthError("Invalid token: signature, api token not found", 403, service_id=service.id)
 
 
-def _auth_by_api_key(auth_token):
+def _auth_by_api_key(auth_token):    
```

Comment:
This will be the eventual validation we enable.  I wrote it now while everything was fresh and then commented it out.  If anyone prefers this to be omitted, just let me know.